### ISSUE TITLE
macbook pro issue

### ISSUE THREAD

**Issue by john7887** [BenSova/Patched-Sur#407]
- Upgrading from: Catalina v10.15.7
- Upgrading to: Big Sur v11.2.1
 - Mac Model: 13" 2010MacBook Pro
 

after install mac os big sir macbook does not boot
![60B98141-046A-458E-BFD9-121F15CC2578](https://user-images.githubusercontent.com/74682554/111190193-4a3aee80-85d0-11eb-9425-3fa43575ae63.jpeg)

**Comment by BenSova** [comment]
What happens if you boot into the EFI Boot on the installer USB?

**Comment by john7887** [comment]
> What happens if you boot into the EFI Boot on the installer USB?

yes, I boot into the EFI boot on the istaller usb, after I push "EFI" macbook switched off, after this I turn on mac via "power" and "option" button again and I installed mac os without clean disk mac os catalina.

**Comment by ozziepadilla** [comment]
> > What happens if you boot into the EFI Boot on the installer USB?
> 
> Yes, I boot into the EFI boot on the istaller usb, after I push "EFI" macbook did shutdown itself, after this I did turn on mac via "power" and "option" button again and I did install mac os without clean disk mac os catalina. After this macbook does not boot up

Hi in the beginning when you run patchet Sur you select UPDATE or CLEAN ? I am Just curious to know

**Comment by BenSova** [comment]
You can't select clean....

**Comment by ozziepadilla** [comment]
> You can't select clean....

I know that Ben ,,😚I am asking if this guy select clean that probably is the issues he is experiencing

**Comment by ozziepadilla** [comment]
> You can't select clean....

I have seen before that screenshot of the issues he got

**Comment by BenSova** [comment]
I mean the patcher doesn't even let you continue. You have to click update otherwise you will never be able to make an installer usb. It's like how you can't even use the patcher with FileVault anymore (v0.0.5+). It'll block you from creating the installer usb.

**Comment by ozziepadilla** [comment]
> I mean the patcher doesn't even let you continue. You have to click update otherwise you will never be able to make an installer usb. It's like how you can't even use the patcher with FileVault anymore (v0.0.5+). It'll block you from creating the installer usb.

I got it  👍

**Comment by john7887** [comment]
> I mean the patcher doesn't even let you continue. You have to click update otherwise you will never be able to make an installer usb. It's like how you can't even use the patcher with FileVault anymore (v0.0.5+). It'll block you from creating the installer usb.

What can I do?

**Comment by ozziepadilla** [comment]
> > I mean the patcher doesn't even let you continue. You have to click update otherwise you will never be able to make an installer usb. It's like how you can't even use the patcher with FileVault anymore (v0.0.5+). It'll block you from creating the installer usb.
> 
> What can I do? How fix my problem?

You know the procedures to reinstall Catalina? Let me know if you don’t know I can guide you step by step

**Comment by Capttain1** [comment]
> > > I mean the patcher doesn't even let you continue. You have to click update otherwise you will never be able to make an installer usb. It's like how you can't even use the patcher with FileVault anymore (v0.0.5+). It'll block you from creating the installer usb.
> > 
> > 
> > What can I do? How fix my problem?
> 
> You know the procedures to reinstall Catalina? Let me know if you don’t know I can guide you step by step

I have the same issues

**Comment by john7887** [comment]
What means Big Sur UI/UX?

**Comment by BenSova** [comment]
I'm just categorizing bugs. Big Sur UI/UX is anything related to how Big Sur acts.

**Comment by john7887** [comment]
> I'm just categorizing bugs. Big Sur UI/UX is anything related to how Big Sur acts.

Fiend! Can you fix it?
